File: samples_generated_code/sample_2/phi4/baseline_code_java/code_row_106.java
public class SalesCalculator {
    public static void main(String[] args) {
        double totalSales = 0;

        double januarySales = 10000;
        double februarySales = 15000;
        double marchSales = 20000;

        totalSales += januarySales;
        totalSales += februarySales;
        totalSales += marchSales;

        System.out.println("Total Sales: " + totalSales);
    }
}

File: samples_generated_code/sample_2/phi4/baseline_code_c/code_row_106.c
#include <stdio.h>

int main() {
    double totalSales = 0;

    double januarySales = 15000;
    double februarySales = 12000;
    double marchSales = 18000;

    totalSales += januarySales;
    totalSales += februarySales;
    totalSales += marchSales;

    printf("Total Sales: %.2f\n", totalSales);

    return 0;
}